modules/earthengine-layers/src/earth-engine-layer.js

```diff
@@ -116,27 +164,36 @@ export default class EarthEngineLayer extends CompositeLayer {
           minZoom,
           maxCacheSize,
           maxCacheByteSize,
+          frame,
 
-          async getTileData({x, y, z}) {
-            const imageUrl = getTileUrl(x, y, z);
-            const image = await load(imageUrl, ImageLoader);
-            return image;
-          },
+          getTileData: options => this.getFilmstripTileData(options),
```

Comment:
We need to make this work with all types of ee objects, not hard code for ImageCollection